app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
```

Comment:
I think these deletions are breaking something on mine. Can somebody try building the project and verify?

Reply:
fixed on new commit